Sentry/src/test/java/io/sentry/util/FileUtilsTest.kt

```diff
@@ -0,0 +1,58 @@
+package io.sentry.util
+
+import org.junit.Test
```

Comment:
```suggestion
import kotlin.test.Test
```

Reply:
Always use the `kotlin.test` package

Reply:
Alright, but can you tell me what's the difference, please?
I saw several tests using `org.junit.Test`, too. Is it a mistake?

Reply:
Yep, they are all wrong, it's just an Alias to be honest, but I'd rather stick to Kotlin packages, so Kotlin first in tests

Reply:
Raise a new issue to replace all `org.junit` please, or change in this PR ^^

Reply:
Same goes with 
`org.junit.Ignore` -> `kotlin.test.Ignore`
`org.junit.Before` -> `kotlin.test.BeforeTest`
`org.junit.After` -> `kotlin.test.AfterTest`
right?